src/guide/project-maintenance.md

```diff
@@ -0,0 +1,66 @@
+# Project Maintenance
+
+Many OpenJDK projects build on top of the JDK source code. When doing this there are a number of common workflows that are dealt with by most project maintainers. For instance updating the code (merging) to bring in the latest changes from the upstream JDK project.
```

Comment:
I agree that a few examples could make it slightly easier to read but I also worry that these examples will rot going forward as the example projects are completed and forgotten. How important would you say this is?

Reply:
Would it help to give examples of the types of Projects that are downstream consumers of the JDK source?  A common case would be forward thinking Projects that exist to think about evolution in a particular area rather than a particular, specific feature.  Amber, Loom, Panama, and Valhalla fall into this category.  All of them are long-running Projects that have been the source of multiple features over several releases.

Reply:
Yes, I think those are good examples. Something like:

> Many OpenJDK projects build on top of the JDK source code. At the time of writing, some examples are Project Valhalla(link), which tries to create a JVM with "value types", and Project Panama(link), which is improving native JVM interoperation. There are a number of workflows that projects like this have to deal with. For instance, updating the project codebase (merging) to bring in the latest changes from the upstream JDK project.

Reply:
Ok. So this is my reasoning in that case; Out of the suggested projects I think project Amber and Valhalla are the most likely to be around for quite some time more. IMO giving descriptions about those projects doesn't add any value to this text but is just likely to rot faster (links are fine).

> Many OpenJDK projects build on top of the JDK source code for instance to produce new language or JVM features, like projects [Amber](https://openjdk.org/projects/amber/) and [Valhalla](https://openjdk.org/projects/valhalla/).

Reply:
I agree with what you're saying, Jesper, probably just links are a better option.

Reply:
Fixed.